src/app/routes/connector-ui/connector-ui.module.ts

```diff
@@ -17,6 +17,7 @@ import {ConnectorUiComponent} from './connector-ui.component';
 import {ContractAgreementPageModule} from './contract-agreement-page/contract-agreement-page.module';
```

Comment:
Changelog missing, this is a bugfix from a product perspective, so let's track it

Reply:
Updated changelog